src/components/navigation-header/style.module.css

```diff
@@ -33,20 +33,36 @@
   list-style: none;
   margin-bottom: 0;
   margin-left: 0;
-  margin-right: 56px;
+  margin-right: 72px;
   margin-top: 0;
 
   & li {
-    padding: 8px 12px;
+    padding: 4px;
 
     & a {
-      color: var(--navigation-header-link-color);
+      border: 1px solid transparent;
+      border-radius: 5px;
+      color: var(--token-color-palette-neutral-400);
+      display: block;
       font-size: 14px;
-      line-height: 16px;
+      font-weight: 500;
+      line-height: 20px;
+      padding: 3px 8px;
 
       &:hover {
-        cursor: pointer;
-        text-decoration: underline;
+        color: var(--token-color-palette-neutral-300);
+      }
```

Comment:
nit: It'd be a nice touch if there was a very short transition on the hover color change here ✨

A .1s makes transition makes it feel fancy 😍

Reply:
Done! Great idea